internal/configs/node/config.go

```diff
@@ -61,6 +65,21 @@ type ConfigType struct {
 	isLeader bool        // whether this node is a leader or not
 	shardID  uint32      // shardID of this node
 	role     Role        // Role of the node
+
+	// TODO(minhdoan): Clean up later.
+	ShardIDString   string
+	StringRole      string
```

Comment:
This is redundant. please clean it up for sure.

Reply:
right now, i'm moved all code around. in the original harmony.go role is used as a string and it's easier for me to keep the same type because refactoring more will make people more difficult to review code. let's wait for the next PR